src/components/SuggestedVideosContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import { StyledSuggestedVideosContainer } from '../components/styles/SuggestedVideosContainer.styled';
+import { useParams } from 'react-router-dom';
+import useFetch from '../hooks/useFetch';
+import SuggestedVideoCard from './SuggestedVideoCard';
+import Alert from 'react-bootstrap/Alert';
+
+const SuggestedVideosContainer = () => {
+  const { id } = useParams();
+  const { loading, error, data } = useFetch(
+    // eslint-disable-next-line no-undef
+    `https://youtube.googleapis.com/youtube/v3/search?part=snippet&maxResults=20&relatedToVideoId=${id}&type=video&key=${process.env.REACT_APP_API_KEY}`,
+    {},
+    id
+  );
+  const suggestedVideos = data
+    ? data.items.filter((video) => video.snippet)
+    : [];
```

Comment:
maybe you can pass a function to the fetch that if it is passed it filters the data as you want so it is returned in the final form you are going to use it